@types/TokenBuildInputArray.d.ts

```diff
@@ -0,0 +1,10 @@
+type TokenBuildInputItem = {
+  // Used as the filename for the generated file (e.g. `dark`.css), extension will be appended
+  outputName: string
```

Comment:
could this be filename instead? or is it used for something else too? Output can mean different things in style dictionary and build process, so filename feels more explicit.

Reply:
Yes, it is used for other stuff sometimes. I was afraid that people would add the extension if we call it filename. So I need just the name, no extension. Would you still name it `filename`?

Reply:
Maybe we could also go for `basename`? https://stackoverflow.com/questions/280767/definition-of-filename

Reply:
I see you changed to outnameName, that's definitely an improvement. Happy to stick with that. I still think filename is clearest IMO, also because in JS, the parse() fn will return `name` as the filename sans exension. Basename also works, but also probably requires some assumed knowledge there?